fineract-provider/src/main/java/org/apache/fineract/batch/command/CommandStrategyProvider.java

```diff
@@ -92,6 +92,8 @@ private void init() {
         this.commandStrategies.put(CommandContext.resource("loans\\/\\d+\\/charges").method("POST").build(), "createChargeCommandStrategy");
         this.commandStrategies
                 .put(CommandContext.resource("loans\\/\\d+\\/charges").method("GET").build(), "collectChargesCommandStrategy");
+        this.commandStrategies
+                .put(CommandContext.resource("loans\\/\\d+\\/transactions").method("POST").build(), "repaymentCommandStrategy");
```

Comment:
Please name the new repayment strategy inline with the existing names. Ex : "repayLoanCommandStrategy"

Reply:
I have made those changes. Thanks